frontend/src/v5/helpers/colors.helper.ts

```diff
@@ -89,13 +89,20 @@ export const rgbGroupColorToHex = ({ opacity, color }: RgbGroupColor): HexGroupC
 
 // Hex converters
```

Comment:
I can still see the bug when on views created in the same session (refreshing seems to fix it)

https://github.com/user-attachments/assets/69f10f8e-8018-4e3f-bd94-4b5433f0a2cc